src/Hooks/useFetch.js

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from "react";
+
+function useFetch(url, options = {}) {
+	const [response, setResponse] = useState(null);
+	const [error, setError] = useState(null);
+	// What is a real use case for isLoading? It seems redundant to me.

```

Comment:
you need some indicator of "we are done fetching". You can check if your API returned a value but what happens if returning nothing is a valid response(quite common)? without some sort of loading indicator you cannot safely show loading spinners and|or loading bars.

I think this part should be outside for the scope of the course but, normally, I don't use a simple "isLoading" flag but something like:

https://usehooks-typescript.com/react-hook/use-fetch 

or 

https://kentcdodds.com/blog/stop-using-isloading-booleans